minesweeper.py

```diff
@@ -279,7 +279,21 @@ def highscore(mins, secs):
             top_10_times.insert(index, converted_to_seconds)  # SAME AS ABOVE
 
 
+# with open("high_scores.json", "w") as high_scores:
+#    json.dump(self.game_time.strftime("%M:%S"), high_scores)
+
+
+def slider_value_to_json():
+    json_dict["height_slider"] = int(height_slider.scale.get())
+    json_dict["width_slider"] = int(width_slider.scale.get())
+    json_dict["difficulty_slider"] = int(difficulty_slider.scale.get())
+    with open("high_scores.json", "w") as sliders:
+        json.dump(json_dict, sliders)
+    # close json file (however that is done...)
```

Comment:
This comment is unnecessary. Because you use `with`, the file is closed automatically.